Plugins/Gource/GourceStart.cs

```diff
@@ -153,7 +153,7 @@ private void linkLabel1_LinkClicked(object sender, LinkLabelLinkClickedEventArgs
 
         private void linkLabel2_LinkClicked(object sender, LinkLabelLinkClickedEventArgs e)
         {
-            Process.Start(@"https://github.com/acaudwell/Gource/blob/master/README");
+            Process.Start(@"https://github.com/acaudwell/Gource#readme");
```

Comment:
I recommend using https://github.com/gitextensions/gitextensions/blob/master/GitUI/OsShellUtil.cs#L45.

Reply:
Hi @vbjay I am not sure how can I use this ? Can you please guide me ?

Reply:
In cases like that, a good habit is to have a look at how it is already used and you will find something like that:
`OsShellUtil.OpenUrlInDefaultBrowser(@"https://github.com/acaudwell/Gource#readme");`

Reply:
Thanks @pmiossec.